astrogrism/simulate/simulate.py

```diff
@@ -1,27 +1,36 @@
 from math import floor
+from os import environ
 from pathlib import Path
+from tarfile import open as tar_open
+from tempfile import gettempdir
 
 from astrogrism import GrismObs
 from astropy.io import fits
 from astropy.wcs import WCS
 from astropy import units as u
+from astropy.utils.data import download_file
 import numpy as np
 
 from specutils import Spectrum1D
-import stsynphot as stsyn
 
-
-def _fake_spectrum():
-    spec_axis = np.linspace(6000, 8000, 10) * u.AA
-    flux = (np.random.randn(len(spec_axis.value)) +
-            10*np.exp(-0.001*(spec_axis.value-6563)**2) +
-            spec_axis.value/500) * u.Jy
-
-    return Spectrum1D(spectral_axis=spec_axis, flux=flux)
+SIM_DATA_DIR = Path(gettempdir()) / "astrogrism" / "simulation_files"
```

Comment:
Not everyone has write access to `/tmp` or want to clutter it with all the TRDS files. You should give user option to configure this.

Reply:
Interesting thought; I had always been relying on `gettempdir` to give me a writeable folder, but never actually looked up what folder it returns. When I looked it up, the docs imply `tempfile.gettempdir` shouldonly return `/tmp` if the user has write access to the folder. If it doesn't, it has a series of different fallbacks, including the current working directory: https://docs.python.org/3/library/tempfile.html#tempfile.gettempdir

Reply:
Ah, good to know about the fallbacks. Still, I think this should be user configurable.

Reply:
True, the user configurable part is a good idea. Maybe in a conf file? And have `gettempdir` as the fallback if none is specified

Reply:
Do you think this will suit your needs? https://docs.astropy.org/en/latest/config/index.html

Reply:
I use it myself, for example at https://github.com/spacetelescope/stsynphot_refactor/blob/master/stsynphot/config.py , FWIW.

Reply:
Well, that looks like exactly what I needed; thanks!!

Reply:
Ended up going for an argument rather than a full conf file. Seemed like overkill for the time being. Might revisit if we find more options being needing to be configurable to the user